codepen/tree-sitter-html
# Html includes

Actual changes:
*  `grammer.js` - the "tribracket" and "tribracket_content" rules
* `corpus/codepen_extensions.txt` - added this testfile
* `.gitignore` - to ignore yarn.lock

To test:
```bash
yarn install
yarn tree-sitter generate && yarn tree-sitter test
```

Tests should pass.  Check out the corpus/codepen_extensions.txt test to see input and expected S-E.
